resources/lib/clientinfo.py

```diff
@@ -39,3 +39,6 @@ def getDeviceId(self):
     def getVersion(self):
         version = xbmcaddon.Addon(id="plugin.video.embycon").getAddonInfo("version")
         return version
+
+    def getClient(self):
+        return 'Kodi EmbyCon'
```

Comment:
are you sure this works?
the issue is that the server is expecting a very particular client name for it to correctly itentify the client type.
from memory it needed to be XBMC but we can probably confirm that with Luke.
based on the main addon it uses "Kodi" so at a minimum it would need to be Kodi I think.
you could confirm this by starting the addon and making sure it identifies correctly in the Emby server main page where it shows connected clients.